flight_log_be/serializers.py

```diff
@@ -1,7 +1,13 @@
 from rest_framework import serializers
 from .models import User
+from .models import Flight
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ['id', 'first_name', 'last_name']
\ No newline at end of file
+        fields = ['first_name', 'last_name']
+
+class FlightSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Flight
+        fields = ['id', 'user', 'date', 'start_location', 'end_location', 'day_hours', 'night_hours', 'aircraft', 'description', 'role']
```

Comment:
In my serializer (to be PR'd later) i've removed the ID attribute from the serializer, and return it outside of the attributes by calling Flight.objects.last().id, but I think that when I put up my PR it will make more sense than I can make in this comment

Reply:
Same! I took "id" out of the flight serializer in my version too, and then am returning it in the response similar to how @KatieLonsdale is doing in the User endpoint.  We can make that change in our PRs.